Docker-Swarm-deployment/single-node/ogc-rs/ogc-rs-stack.yaml

```diff
@@ -2,7 +2,7 @@ version: "3.9"
 
 services:
   ogc-rs:
-    image: ghcr.io/datakaveri/geoserver-dev:1.0.0-alpha-96bb4be
+    image: ghcr.io/datakaveri/geoserver-dev:1.0.0-alpha-e8e62ae
```

Comment:
geoserver is having only alpha tag images @abhi4578

Reply:
Should be okay for now.  
+ cc @karun-singh